app/views/spree/order_mailer/_payment.html.haml

```diff
@@ -3,12 +3,15 @@
     - if @order.paid?
       = t :email_payment_paid
     - else
-      = t :email_payment_not_paid
+      - if @order.outstanding_balance?
+        = @order.outstanding_balance < 0 ? t(:credit_owed) : t(:balance_due)
```

Comment:
is this code not already somewhere else? this is logic that should not be in a view but in a service, can we extract to service and call it from here?